### PR TITLE
Gamepad: test interfaces are in insecure context

### PR DESCRIPTION
**Pull request by marcoscaceres**
https://github.com/w3c/gamepad/pull/194

### CODE REVIEW

**Comment by reillyeon** [comment]
Can we just delete this test and rename `idlharness.https.window.js` to `idlharness.http.window.js`? The existing test is more thorough and doesn't require maintaining hand-written checks.

**Comment by reillyeon** [comment]
When working with GitHub PRs please avoid doing force pushes. It breaks the relation chain between revisions of the PR and makes review harder. You can squash the branch history into a single commit when merging the PR.

File: gamepad/gamepad-secure-context.html [hunk 1]
```diff
@@ -5,21 +5,13 @@
...
-    assert_equals(typeof(GamepadButton), "undefined",
-                  "GamepadButton type is undefined");
-    assert_equals(typeof(GamepadEvent), "undefined",
-                  "GamepadEvent type is undefined");
-  }, "Gamepad API is not exposed in non-secure contexts");
+    assert_true("Gamepad" in window, "Gamepad in window");
+    assert_true("GamepadButton" in window, "GamepadButton in window");
+    assert_true("GamepadEvent" in window, "GamepadButton in window");
+    assert_true("GamepadHapticActuator" in window, "GamepadButton in window");
+  }, "Gamepad API and interfaces are exposed in non-secure contexts");
```
Comment (reillyeon) [inline gamepad/gamepad-secure-context.html:16]: What do you think about deleting this file entirely?

**Review by reillyeon — CHANGES_REQUESTED** [review]

File: gamepad/gamepad-secure-context.html (hunk 1 above)
Comment (marcoscaceres) [reply gamepad/gamepad-secure-context.html:16]: yeah... I was thinking the same... [idlharness.window.js](https://github.com/web-platform-tests/wpt/pull/44691/files#diff-a788f3dd179ecc15a731b46a982ce5ca0a737a67e92402a85a459dee75507ea7) does the same thing.

**Review by reillyeon — APPROVED** [review]

**Review by saschanaz — APPROVED** [review]
Please resolve conflict, otherwise LGTM (removing secure context and doing non-https idlharness test sounds good, should have done the same.)

**Pull request merged**